sdk/communication/Azure.Communication.Chat/tests/Infrastructure/ChatLiveTestBase.cs

```diff
@@ -20,7 +20,7 @@ protected CommunicationIdentityClient CreateInstrumentedCommunicationIdentityCli
             => InstrumentClient(
                 new CommunicationIdentityClient(
                     TestEnvironment.LiveTestDynamicConnectionString,
-                    InstrumentClientOptions(new CommunicationIdentityClientOptions())));
+                    InstrumentClientOptions(new CommunicationIdentityClientOptions(CommunicationIdentityClientOptions.ServiceVersion.V2021_03_07))));
```

Comment:
Curious why we need to add the service version here for Chat and NetworkTraversal?

Reply:
We want Chat and NetworkTraversal to reference the GA stable version of identity. Also updating to the private preview version is causing some of their tests to fail due to the recordings expecting a different path and I didn't want to be rerecording all of their tests files when its not owned by us.

Reply:
Ideally everyone should by default use the stable GA version, and for the one that we are private previewing we should have a need for optional parameters.